treeherder/webapp/api/logslice.py

```diff
@@ -38,8 +39,8 @@ def list(self, request, project, jm):
         Receives a line range and job_id and returns those lines
         """
         job_id = request.QUERY_PARAMS.get("job_id")
-
-        log = jm.get_log_references(job_id)
+        log_name = request.QUERY_PARAMS.get("name", "builds-4h")
```

Comment:
I know the 'name' is set to "build-4h" in the logs_url when we ingest it currently (https://github.com/mozilla/treeherder-service/blob/e9780bf1a0eb71fc5ea2f9e5fb847dcf72a4aed0/treeherder/etl/buildapi.py#L172) - however I think this isn't a great name, and as such we should use say "buildbot_text_log" or "text_log" (which will match the new "text_log_summary" name of the artifact that's derived from it).

Normally I'd say this was fodder for another bug, but once we land this (and the corresponding UI part), then we have even more pieces to change when we do change the name. How about having the REST API use the "new" name, and then converting it here to "build-4h" temporarily, until a later bug makes the change to what we put in the DB? (We'll need to have a transition period where the API looks for both names, but that should be simple enough in this future bug).